### PR TITLE
Change text/style on app-testing-component

### PR DESCRIPTION
**Pull request by saleazer**
Another small PR to test webhooks

### CODE REVIEW

File: src/app/app-testing/app-testing.component.html [hunk 1]
```diff
@@ -1 +1 @@
-<p>app-testing works!</p>
+<p>This is the app-testing component!</p>
```
Comment (spacekat629) [inline src/app/app-testing/app-testing.component.html:1]: This could be better, just sayin'

File: src/app/app-testing/app-testing.component.css [hunk 2]
```diff
@@ -0,0 +1,4 @@
+p {
+    color: red;
```
Comment (spacekat629) [inline src/app/app-testing/app-testing.component.css:2]: We should add colors to a variables file instead

Comment (saleazer) [reply src/app/app-testing/app-testing.component.css:2]: Understood

File: src/app/app-testing/app-testing.component.html (hunk 1 above)
Comment (saleazer) [reply src/app/app-testing/app-testing.component.html:1]: Yeah, yeah...

**Pull request merged**